contents/docs/product-analytics/identify.mdx

```diff
@@ -18,6 +18,11 @@ export const mergeRestrictionsDark = "https://res.cloudinary.com/dmukukwp6/image
 export const splitUsersLight = "https://res.cloudinary.com/dmukukwp6/video/upload/v1710055416/posthog.com/contents/images/docs/data/identify/how-to-split-users-light-mode.mp4"
 export const splitUsersDark = "https://res.cloudinary.com/dmukukwp6/video/upload/v1710055416/posthog.com/contents/images/docs/data/identify/how-to-split-users-dark-mode.mp4"
 
+<iframe
```

Comment:
Rather than have the first on the page, would it be better to have it after IdentifyIntro? I think having text first and then the video might be better (with video first, people might think the video is the only thing on the page), but don't feel strongly.

Reply:
will try it!

Reply:
the intro is quite long so will keep as is